src/main/java/com/minecolonies/blocks/AbstractBlockHut.java

```diff
@@ -77,23 +76,75 @@ private void initBlock()
 
     @NotNull
     @Override
-    public TileEntity createNewTileEntity(World world, int meta)
+    public TileEntity createNewTileEntity(final World world, final int meta)
     {
         //Creates a tile entity for our building
         return new TileEntityColonyBuilding();
     }
 
     @Override
-    public int getMetaFromState(@NotNull IBlockState state)
+    public boolean isFullBlock(final IBlockState state)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Don't override a deprecated method or explicitly mark it as "@Deprecated". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)